mesonbuild/mtest.py

```diff
@@ -136,20 +136,20 @@ def returncode_to_status(retcode: int) -> str:
             signame = signal.Signals(signum).name
         except ValueError:
             signame = 'SIGinvalid'
-        return '(killed by signal %d %s)' % (signum, signame)
+        return '(killed by signal {0} {1})'.format(signum, signame)
 
     if retcode <= 128:
-        return '(exit status %d)' % (retcode,)
+        return '(exit status {0})'.format((retcode,))
 
     signum = retcode - 128
     try:
         signame = signal.Signals(signum).name
     except ValueError:
         signame = 'SIGinvalid'
-    return '(exit status %d or signal %d %s)' % (retcode, signum, signame)
+    return '(exit status {0} or signal {1} {2})'.format(retcode, signum, signame)
 
 def env_tuple_to_str(env: typing.Iterable[typing.Tuple[str, str]]) -> str:
-    return ''.join(["%s='%s' " % (k, v) for k, v in env])
+    return ''.join(["{0}='{1}' ".format((k, v) for k, v in env)])
```

Comment:
you don't want the parens around `k, v` here, you're actually passing a tuple to str.format instead of two scalar values.

Reply:
Just reverted this change.